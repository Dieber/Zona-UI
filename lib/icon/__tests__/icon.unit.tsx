import * as renderer from 'react-test-renderer'

import React from 'react'
import Icon from '../icon'

describe('Icon test', () => {
  it('is a Icon', () => {
    const json = renderer.create(<Icon/>).toJSON()
    expect(json).toMatchSnapshot()
  })
})