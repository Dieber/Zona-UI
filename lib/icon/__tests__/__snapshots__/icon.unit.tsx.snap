// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon test is a Icon 1`] = `
<svg
  aria-hidden="true"
  className=""
>
  <use
    xlinkHref="#icon-undefined"
  />
</svg>
`;
